scaffolds/src/main/kotlin/com/decathlon/vitamin/compose/scaffolds/TabActionUi.kt

```diff
@@ -0,0 +1,13 @@
+package com.decathlon.vitamin.compose.scaffolds
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.graphics.vector.ImageVector
+
+@Stable
+@Immutable
+data class TabActionUi(
```

Comment:
No content description for tabs?

Reply:
In the actual contract, you are forced to give a label for the tab. So content description isn't necessary.